src/devices/tuya.ts

```diff
@@ -4611,6 +4611,25 @@ export const definitions: DefinitionWithExtend[] = [
             tuya.whitelabel("Nova Digital", "CMR-1", "Roller Blind Motor", ["_TZE200_9vpe3fl1"]),
         ],
     },
+    {
+        fingerprint: tuya.fingerprint("TS0601", ["_TZE200_a8z0g46u", "_TZE204_a8z0g46u"]),
+        model: "TS0601",
```

Comment:
What is the model number written on device or product page?

Reply:
There are many similar products and I think they are using the Tuya Zigbee module to make such window openers. There companies are also using Xiaomi Bluetooth modules for integrating to Xiaomi Home. 

I contacted the vendor of the product, he told me they call it "YY-LT500". Is it OK to use "YY-LT500" or should we name it in a more generic way?

Reply:
The product has printed model on it. It is **"YY-LT500"**.  I think we should use this name. 

I also disassembled the product and it is using [ZTU-IPEX Zigbee module](https://developer.tuya.com/en/docs/Document/ztuipex-module-datasheet?id=Kamwy20nl3rr3).

Reply:
Changed model name, please review, thanks!